### PR TITLE
feat: initialized and added home

### PR DESCRIPTION
**Pull request by aryan-13**
- Converted the homepage into React components
- Added styling

### CODE REVIEW

**Review by bajajcodes — COMMENTED** [review]
- Good applied use of CSS.
- In base/utils.css class named `centered` can be renamed to 
  more meaningful name.
- In navbar.jsx can use `Link` in place of `a` jsx element.
- Apart from logo image, can use CDN for other images.
- Will be using your CSS code as reference for my code.

File: src/css/layout/filterbar.css [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+  
+  /* Show the indicator (dot/circle) when checked */
+  .radio-list input:checked ~ .radio:after {
+    display: block;
+  }
+  
+  /* Style the indicator (dot/circle) */
+  .radio-list .radio:after {
+       top: 5px;
+      left: 5.3px;
```
Comment (Ronak-Mutha) [inline src/css/layout/filterbar.css]: Any particular reason to use decimal pixel value?

File: src/css/layout/horz-cards.css [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+    .wrapper {
+        flex-direction: column;
+        width: 100%;
+    }
+
+    .cart-card-box,
+    .cart-total {
+        width: 100%;
+    }
+
```
Comment (Ronak-Mutha) [inline src/css/layout/horz-cards.css:100]: Multiple blank spaces. Please remove when refactoring using code formatter.

**Review by Ronak-Mutha — COMMENTED** [review]
Try to include suggested changes whenever possible. Also remove unnecessary decimal pixel values, also considering that you might add router later not pointing out anchor related things.
Apart from that, code looks good to merge.

File: src/css/layout/horz-cards.css (hunk 2 above)
Comment (aryan-13) [reply src/css/layout/horz-cards.css:100]: Okay

File: src/css/layout/filterbar.css (hunk 1 above)
Comment (aryan-13) [reply src/css/layout/filterbar.css]: It is for the placement of the radio button

**Pull request merged**